packages/dds/tree/src/test/tree/anchorSet.spec.ts

```diff
@@ -127,7 +127,9 @@ describe("AnchorSet", () => {
 		assert.throws(() => anchors.locate(anchor3));
 	});
 
-	it("can rebase over move", () => {
+	// TODO: Fix bug in move handling.
+	// See https://github.com/microsoft/FluidFramework/pull/14358 and https://dev.azure.com/fluidframework/internal/_workitems/edit/3559
```

Comment:
done.

Reply:
Note that we already had an integration test, but I have now added a unit test.

Reply:
Also note that its unclear why this test broke in my change, but its testing move which is known to be incorrectly implemented, so I assume my changs the symptoms of this incorrect implementation probably broke the test (or fixed it and the test was incorrect). Regardless this test needs to be followed up on when move is fixed (fixing it is easier than debugging this test). The change I made to anchorset that impact this are due to the optimization of using internalizePath, which should have no impact on correctness if moveOut move the contents eargerly  instead of assuming they are in the same tree location later.

Reply:
I have now also added tests for internalizePath